pkg/controller/executors/node_lookup.go

```diff
@@ -14,24 +14,6 @@ type NodeLookup interface {
 	GetNodeExecutionStatus(ctx context.Context, id v1alpha1.NodeID) v1alpha1.ExecutableNodeStatus
 }
 
-// Implements a de-generate case of NodeLookup, where only one Node is always looked up
```

Comment:
Why are we removing this?

Reply:
we dont use it, we can add it when we need it